blockchain/stakeversion.go

```diff
@@ -82,7 +82,7 @@ func (b *BlockChain) isStakeMajorityVersion(minVer uint32, prevNode *blockNode)
 	// Walk blockchain backwards to calculate version.
 	node := b.findStakeVersionPriorNode(prevNode)
 	if node == nil {
-		return 0 >= minVer
+		return true
```

Comment:
This isn't the same logic.

```
minVer=0: 0 >= 0 == true
minVer=1: 0 >= 1 == false
```

It should be return `minVer == 0` to avoid the lint error and retain the same logic.